src/Illuminate/Http/Request.php

```diff
@@ -459,7 +459,7 @@ protected function convertUploadedFiles(array $files)
      *
      * @param  string  $key
      * @param  mixed  $default
-     * @return \Symfony\Component\HttpFoundation\File\UploadedFile|array|null
+     * @return \Illuminate\Http\UploadedFile|array|null
```

Comment:
The `file()` method just pulls a key from `allFiles()` which [returns an array](https://github.com/laravel/framework/blob/45617b0ed1613998c012a0ed507b02a4a0841260/src/Illuminate/Http/Request.php#L429) of converted uploaded files.

You can see in the `convertUploadedFiles()` method, it [converts](https://github.com/laravel/framework/blob/45617b0ed1613998c012a0ed507b02a4a0841260/src/Illuminate/Http/Request.php#L444) each Symphony UploadedFile object to an Illuminate one via `\Illuminate\Http\UploadedFile::createFromBase()`.